src/AppBundle/Tests/Controller/PostControllerApiTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace AppBundle\Tests\Controller;
+
+use Lakion\ApiTestCase\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+class PostControllerApiTest extends JsonApiTestCase
```

Comment:
I think this class can be sad that it's destiny is to be removed :( (but :+1: for explanation)